_layouts/roadmap.html

```diff
@@ -29,7 +29,7 @@ <h1 class="title">Particl Project<br />Roadmap</h1>
           </div>
           <h3 class="title">Market Listing Governance</h3>
           <div class="desc">
-            <p>The Particl Marketplace is being built to be self-sustaining, self-governing and completely decentralized. Governance is a huge piece of the puzzle that the team will be focused on developing to protect privacy and protect users.</p>
+            <p>Currently under development and nearing testnet, the Marketplace Governance system will allow users to self-moderate the content listed on Particl Marketplace. This system is entirely controlled by the community and no Particl team member or staff has any authority over the content of the marketplace.</p>
```

Comment:
Can you please rewrite this, so it's not relative to time? That was probably not clear, so let me explain :)
These roadmap texts should be more descriptions of the items than updates on where we're currently regarding development (that's what the tags above `completed` `in progress` etc. are for). If we'd write it "timing-neutral", we won't have to rewrite it each time some progress is made.. Hope I made myself a bit clearer :)

In short, it means to get rid of the first sentence:
> Currently under development and nearing testnet,